mapreader/download/tileserver_access.py

```diff
@@ -172,11 +172,9 @@ def modify_metadata(
         Parameters
         ----------
         remove_image_ids : list of str, optional
-            List of image IDs to remove from the metadata (default is an empty
-            list, ``[]``).
+            List of image IDs to remove from the metadata (default is an empty list, ``[]``).
         only_keep_image_ids : list of str, optional
-            List of image IDs to keep in the metadata (default is an empty
-            list, ``[]``).
+            List of image IDs to keep in the metadata (default is an empty list, ``[]``).
 
```

Comment:
I realise that this change is only a formatting change in the docstring, but I'm not quite sure of the logic of this function.

* I can't see if this is used anywhere. If it is only used as a helper function by users, then ideally, there should be an example in one of the notebooks.
* If it's not used at all then it should be removed.

If it is used, then how does it handle edge cases? Eg
* What happens if both `remove_image_ids` and `only_keep_image_ids` are empty? Does everything get removed or does everything get kept? Which takes precedence?
* What happens if both lists contain the same IDs? Does an error/exception get raised to the user?

Either way, these questions should get moved to a separate issue, rather than trying to resolve them here.

Reply:
Made this as an issue (#161).